src/styles.css

```diff
@@ -124,6 +124,83 @@ ul {
   justify-content: space-evenly;
 }
 
+select {
+  cursor: pointer;
+}
+
+.dropDropMenu {
+  display: flex;
+  justify-content: space-between;
+  width: 35%;
+}
+
+.quantityInput {
+  width: 3.5vw;
+}
+
+.selectMenu {
+  width: 10vw;
+}
+
+.pantry {
+  background: #E3D3D3;
+}
+
+.pantryContents {
+  height: 100vh;
+  width: 100vw;
+  display: flex;
+  justify-content: center;
+  margin-top: 30px;
+  flex-direction: column;
+  align-items: center;
+}
+
+.pantryItems {
+  display: flex;
+  width: 32vw;
+  height: 100vh;
+  justify-content: space-evenly;
+  margin-top: 15px;
+}
+
+.pantryHeading {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 24vw;
+}
+
+.table {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 30vw;
+  align-items: center;
+}
+
+.addButton {
+  font-size: 1em;
+}
+
+.boxI {
+  border: 2px solid white;
+  height: 4vh;
+  width: 19vw;
+}
+
+.boxA {
+  border: 2px solid white;
+  height: 4vh;
+  width: 5vw;
+}
+
+.miniButtons {
```

Comment:
if you add the following code here, it will align the "-" and "+" perfectly in the middle regardless of window size. Right now, they seem to lean towards the bottom of the button.
```
display: flex;
align-items: center;
justify-content: center;
```